tronweb/css/tronweb.less

```diff
@@ -792,6 +792,34 @@ div.refresh-view span {
     .success;
 }
 
+.enabled,
+.up,
```

Comment:
Let's combine this block with the one above.

Reply:
good idea